provision-enterprise-subsidy.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+set -x
+
+. scripts/colors.sh
+
+name="enterprise-subsidy"
+port="18280"
+
+docker compose up -d lms
+docker compose up -d ${name}
+
+# Run migrations
+echo -e "${GREEN}Running migrations for ${name}...${NC}"
+docker exec -t edx.devstack.enterprise-subsidy bash -c "cd /edx/app/${name}/ && make migrate"
```

Comment:
You should use `docker compose exec` and then you don't have to provide the entire container name and instead can only write `enterprsie-subsidy`. You can follow any other provision script for the format.